features/promotion/receiving_discount/receiving_discounts_with_product_minimum_price_specified.feature

```diff
@@ -8,7 +8,6 @@ Feature: Receiving discounts with product minimum price specified
         Given the store operates on a single channel in "United States"
         And the store ships everywhere for Free
         And the store allows paying Offline
-        And I am a logged in customer
```

Comment:
Why

Reply:
as this scenario is described for visitor above